### PR TITLE
Update parameters.rst

### PR DESCRIPTION
**Pull request by lge0303**
Add in situ visualization control parameters
This should be the same PR as #980. Please close and ignore PR980

### CODE REVIEW

**Comment by MaxThevenet** [comment]
Thanks, looks good, but could you remove end-of-line whitespaces in 
```
Docs/source/running_cpp/parameters.rst
```

**Comment by lge0303** [comment]
Where are those end-of-line whitespaces? Could you point them? Thanks!

**Comment by MaxThevenet** [comment]
They are responsible for the failing `source / style` check, see below.
More information at https://github.com/ECP-WarpX/WarpX/pull/990/checks?check_run_id=651166405 where you can expand the `End-of-Line whitespaces` tab. You will find commands that you can copy-paste to your terminal to fix them. Otherwise, all editors propose the option to delete end-of-line whitespaces. Unfortunately I don't know where they are exactly.

**Comment by lge0303** [comment]
@MaxThevenet I fixed the end-of-line whitespaces issues on my local computer terminal, and commit and push back to the github.  Hope it is OK now :-) Thanks a lot for your help!

**Comment by MaxThevenet** [comment]
Looks good, thanks!

File: Docs/source/running_cpp/parameters.rst [hunk 1]
```diff
@@ -1507,6 +1507,24 @@ Reduced Diagnostics
     The separator between row values in the output file.
     The default separator is a whitespace.
 
+In-situ visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+Besides the diagnostics described above, WarpX has in-situ visualization capabilities.
+This is controlled by the following option(s):
+
+* ``insitu.int`` (`integer`; 0 by default)
```
Comment (ax3l) [inline Docs/source/running_cpp/parameters.rst:1516]: As a follow-up PR, can we unify `int` and `start` with the syntax introduced by @NeilZaim in #925, please?

Comment (ax3l) [reply Docs/source/running_cpp/parameters.rst:1516]: As another follow-up PR, can we rename the generic `insitu.<...>` name to the more descriptive `sensei.<...>` please?

Comment (ax3l) [reply Docs/source/running_cpp/parameters.rst:1516]: cc @lge0303

**Pull request merged**